### PR TITLE
Integration test for migration & reverse migration between two clusters

### PR DESCRIPTION
**Pull request by Rohit-PX**
Signed-off-by: Rohit-PX <rohit@portworx.com>

**What type of PR is this?**
integration-test

**What this PR does / why we need it**:
Test migration to and from clusters

**Does this PR change a user-facing CRD or CLI?**:
no

**Is a release note needed?**:
no

**Does this change need to be cherry-picked to a release branch?**:
2.3

### CODE REVIEW

**Comment by Rohit-PX** [comment]
Running here:  https://jenkins.portworx.co/job/Stork/view/Stork%20master%20Dashboard/job/stork-dev-test-2/23/console

File: test/integration_test/common_test.go [hunk 1]
```diff
@@ -53,8 +53,8 @@ const (
 	remotePairName        = "remoteclusterpair"
 	remoteConfig          = "remoteconfigmap"
 	specDir               = "./specs"
-	pairFilePath          = "./specs/cluster-pair"
-	pairFileName          = pairFilePath + "/cluster-pair.yaml"
+	defaultClusterPairDir = "cluster-pair"
+	pairFileName          = "/cluster-pair.yaml"
```
Comment (disrani-px) [inline test/integration_test/common_test.go]:
```suggestion
	pairFileName          = "cluster-pair.yaml"
```

File: test/integration_test/common_test.go [hunk 2]
```diff
@@ -338,13 +338,14 @@ func setRemoteConfig(kubeConfig string) error {
 	return nil
 }
 
-func createClusterPair(pairInfo map[string]string, skipStorage bool) error {
-	err := os.MkdirAll(pairFilePath, 0777)
+func createClusterPair(pairInfo map[string]string, skipStorage, resetConfig bool, clusterPairDir string) error {
+	err := os.MkdirAll(specDir+"/"+clusterPairDir, 0777)
```
Comment (disrani-px) [inline test/integration_test/common_test.go]:
```suggestion
	err := os.MkdirAll(path.Join(specDir, clusterPairDir), 0777)
```

File: test/integration_test/common_test.go [hunk 3]
```diff
@@ -338,13 +338,14 @@ func setRemoteConfig(kubeConfig string) error {
...
 	if err != nil {
-		logrus.Errorf("Unable to make directory (%v) for cluster pair spec: %v", pairFilePath, err)
+		logrus.Errorf("Unable to make directory (%v) for cluster pair spec: %v", specDir+"/"+clusterPairDir, err)
```
Comment (disrani-px) [inline test/integration_test/common_test.go]: Use `path.Join()` for filename

File: test/integration_test/common_test.go [hunk 4]
```diff
@@ -338,13 +338,14 @@ func setRemoteConfig(kubeConfig string) error {
...
 		return err
 	}
-	pairFile, err := os.Create(pairFileName)
+	clusterPairFileName := specDir + "/" + clusterPairDir + pairFileName
```
Comment (disrani-px) [inline test/integration_test/common_test.go]:
```suggestion
	clusterPairFileName := path.Join(specDir, clusterPairDir, pairFileName)
```

File: test/integration_test/common_test.go (hunk 3 above)
Comment (Rohit-PX) [reply test/integration_test/common_test.go]: Fixed.

**Pull request merged**